app/views/donors/show.html.erb

```diff
@@ -18,20 +18,30 @@
   <table class="table">
     <thead class="thead-default">
       <tr>
-        <th>Date</th>
+        <th>Date Last Modified</th>
```

Comment:
Hm. We still want to know when the donation was made. Seeing changes is part of a bigger, cross cutting concern of auditing. 😀